services/crosschainconnector/ethereum/adapter/node_access.go

```diff
@@ -0,0 +1,28 @@
+package adapter
+
+import (
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+type EthereumNodeConnector struct {
```

Comment:
Let's rename and remove the word `Connector`

Maybe `EthereumAdapter`.. the word `connector` is overused on the service level